.github/workflows/docs.yml

```diff
@@ -26,6 +29,7 @@ jobs:
       - name: Build docs
         uses: dls-controls/pipenv-run-action@v1
         with:
+          python-version: "3.8"
```

Comment:
I don't see why the matrix is defined above and the python-version is defined here. Usually the `python-version` is defined as `${{ matrix.python }}`, to run it on whatever list is defined in the matrix?

Reply:
Yes I agree, it didn't work when I defined it with the `${{ matrix.python }}` definition so I just hardcoded the `3.8` to get this pipeline to pass. Seeing as the pipelines are based on the pipenv installation I decided not to waste too much time getting this perfect as it will need updating as soon as we apply the skeleton anyway.

Reply:
I did change the ordering of `with` arguments there so I could try putting it back, but as I said, I was just looking for a quick way to get it to pass.